packages/manager/modules/bm-server-components/src/general-info-tile/controller.js

```diff
@@ -0,0 +1,44 @@
+import isFunction from 'lodash/isFunction';
+
+export default class BmServerComponentsServerGeneralInfoTileController {
+  /* @ngInject */
+  constructor($translate, coreConfig) {
+    this.$translate = $translate;
+    this.region = coreConfig.getRegion();
+  }
+
+  showNameActions() {
+    return this.region !== 'US' && isFunction(this.onEditName);
```

Comment:
Could be feature flipped maybe ?

Reply:
Yes but not part of this refactoring work.